src/components/Login/Login.js

```diff
@@ -1,3 +1,83 @@
+import { useState } from 'react';
+import { KAKAO_AUTH_URL } from './OAuth';
+import { S } from './LoginStyled';
+import { AiOutlineClose } from 'react-icons/ai';
+import { FiMail } from 'react-icons/fi';

```

Comment:
```suggestion
import { AiOutlineClose,FiMail } from 'react-icons/ai';
```
같은 패키지에서 import 한 컴포넌트라면 이렇게 한줄로 묶을 수 있습니다

Reply:
주소가 달라서 나눠놨습니다